model/service.go

```diff
@@ -177,6 +177,13 @@ type ServiceDiscovery interface {
 
 	// HostInstances lists service instances for a given set of IPv4 addresses.
 	HostInstances(addrs map[string]bool) []*ServiceInstance
+
+	// Gets all the Istio service accounts mapped from service hostname, in istio identity format.
+	// For example,
+	// GetIstioServiceAccounts(catalog.myservice.com) -->
+	//      --> [istio:accountfoo.mynamespace, istio:accountbar.mynamespace]
+	// GetIstioServiceAccounts(backend.myservice.com) --> [istio:backendaccount.mynamespace]
+	GetIstioServiceAccounts(hostname string) []string
```

Comment:
Shouldn't this be in platform and not the model ? The idea of service accounts is not platform neutral. The concept of tls certs per service is generic enough.